source/gltf/material.js

```diff
@@ -770,6 +770,7 @@ class KHR_materials_specular extends GltfObject {
         {
             const specularTexture = new gltfTextureInfo();
             specularTexture.fromJson(jsonSpecular.specularTexture);
+            specularTexture.linear = false;
```

Comment:
Should this apply to `specularColorTexture` instead?

Reply:
Yes, seemed to have scrolled to the wrong position.